modules/sdk-lib-mpc/src/tss/ecdsa/paillierproof.ts

```diff
@@ -9,12 +9,12 @@ import { primesSmallerThan319567 } from './primes';
 const k = 128;
 // eprint.iacr.org/2018/057.pdf#page6 section 5
 // https://github.com/BitGo/BitGoJS/pull/3502#discussion_r1203070392
-const alpha = 319567;
+export const alpha = 319567;
 export const m = Math.ceil(k / Math.log2(alpha));
 
 /**
- * Generate a set of challenges $p$ for a given pallier public key modulus $n$.
- * @param n - pallier public key modulus
+ * Generate a set of challenges $p$ for a given paillier public key modulus $n$.
+ * @param n - paillier public key modulus
```

Comment:
Arguably the log here shouldn't hardcode 3072 but interpolate `minModulusBitLength`

Reply:
fixed

Reply:
fixed